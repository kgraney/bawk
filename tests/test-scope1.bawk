
a = 1000;
b = 200;

def foo() {
	a = 10 + 100;
	print(a);
	print(b);
}

print(a);
foo();
print(a);
print(b);

