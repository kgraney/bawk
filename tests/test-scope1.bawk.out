1000
110
200
1000
200
